src/hexo/themes/sonarwhal/source/core/css/scan-results.css

```diff
@@ -26,17 +26,24 @@
     text-transform: capitalize;
 }
 
-.scan-overview__status.analyzing {
+.analyzing {
     animation: fade 3s linear infinite;
 }
 
+@-webkit-keyframes fade {
```

Comment:
@ststimac are you sure we need the `-webkit` prefix. We are in chrome 62 and it has been supported without prefixes for about 20 versions

Reply:
Yes we can delete. caniuse.com is showing full support.  And this particular animation is an enhancement so in older browsers it should be fine. It will still say analyzing just no fade in/out